packages/transformer/package.json

```diff
@@ -39,7 +39,8 @@
     "@pixi/display": "^7.0.0",
     "@pixi/events": "^7.0.0",
     "@pixi/graphics": "^7.0.0",
-    "@pixi/math": "^7.0.0"
+    "@pixi/math": "^7.0.0",
+    "pixi.js": "^7.2.0"
```

Comment:
Can we avoid installing `pixi.js` and instead keep the individual packages? There are no changes to imports, so this seems unwarranted :-)

Reply:
Sure, I have dropped the second commit.